keyboards/nyquist/keymaps/notmarkmiranda/keymap.c

```diff
@@ -0,0 +1,232 @@
+#include "nyquist.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
```

Comment:
You may want to consider using an enum (like with the keycodes below) to make management of layers easier.

This isn't required, it's just a suggestion.